graphics/src/DelaunayTriangulation_TEST.cc

```diff
@@ -63,18 +57,39 @@ TEST_F(GTSMeshUtils, DelaunayTriangulation)
   edges.push_back(gz::math::Vector2i(6, 7));
   edges.push_back(gz::math::Vector2i(7, 4));
 
-  auto mesh = std::make_unique<common::Mesh>();
+  auto mesh = std::make_unique<gz::common::Mesh>();
   mesh->SetName("extruded");
-  common::SubMesh subMesh;
+  gz::common::SubMesh subMesh;
 
-  bool result = common::GTSMeshUtils::DelaunayTriangulation(vertices,
-                                                            edges,
-                                                            &subMesh);
+  bool result = gz::common::DelaunayTriangulation(vertices,
+                                                  edges,
+                                                  &subMesh);
   EXPECT_TRUE(result);
 
   // same as number of vertices in the path
   EXPECT_EQ(subMesh.VertexCount(), 8u);
 
   // there should be 8 triangles.
   EXPECT_EQ(subMesh.IndexCount() / 3u, 8u);
+
+  // verify that triangles have clockwise winding
+  for (int t = 0; t < subMesh.IndexCount() / 3u; ++t)
```

Comment:
```suggestion
  for (unsigned int t = 0; t < subMesh.IndexCount() / 3u; ++t)
```
apologies I missed the compiler warning in #624